.yamato/_triggers.yml

```diff
@@ -22,9 +22,10 @@ pull_request_trigger:
   dependencies:
 {% for project in projects -%}
     - .yamato/project-tests.yml#validate_{{ project.packages.first.name }}_{{ test_platforms.first.name }}_{{ test_editors.first }}
```

Comment:
We're running validation on MLAPI with this line - feels like we should do it for UTP as well?